hm2task2.c

```diff
@@ -0,0 +1,53 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+
+float expLinear(float number, int power) 
+{
+    if (power == 0) 
+        return 1;

```

Comment:
```suggestion
    if (power == 0)
    {
        return 1;
    }
```
Хоть си и подобные ему языки пи позволяют не заключать один оператор в фигурные скобки в данном случае,  принято таки заключать (исправь все такие ситуации в этом решении).